exercises/ruby-1-b/ruby_1_b_example.rb

```diff
@@ -1,12 +1,95 @@
-VOWELS = %w{a e i o u}
-def word_with_most_vowels(sentence)
-  words = sentence.split(" ")
-  word_vowels = words.map do |word|
-    [
-      word,
-      word.downcase.chars.select{|l|VOWELS.include?(l)}.count
-    ]
-  end
-  word_vowels.sort_by {|w,c| -c}.first.first
+class Library
```

Comment:
I'm wondering if this example is perhaps already took bit for being solved in 10-15 minutes? There is already lots going on. We can perhaps strip things out a bit and move additional requirements to the `ruby-2-b` part?

Reply:
Thanks for the review! I agree that there is a bit too much stuff in this part. What do you think about this division:

Part 1:
- find
- add_copies
- copies (new method, returns number of copies)

Part 2:
- borrow
- return
- borrowed
- (perhaps something else)